tangelo/algorithms/variational/iqcc_ilc_solver.py

```diff
@@ -29,7 +29,10 @@
     2. R. A. Lang, I. G. Ryabinkin, and A. F. Izmaylov.
         J. Chem. Theory Comput. 2021, 17, 1, 66–78.
 """
+from typing import List, Union, Optional
 
+from tangelo import SecondQuantizedMolecule
```

Comment:
I think we are importing those two just to enable typing. I'm trying to understand if there are unwanted consequences to these imports.

It seems perfectly acceptable for our algorithm to depend of these data-types, and its execution would require them anyway I believe (we eventually will have a QubitOperator, and as for SecondQuantizedMolecule it should hopefully not require pyscf/psi4 if the user is actually not throwing in a molecule but directly a qubit Hamiltonian at the algorithm) . Is that correct ?

If yes you can resolve0